benchmarks/common.py

```diff
@@ -323,10 +325,12 @@ def speedup_experiment(args, model_iter_fn, model, example_inputs):
     pvalue = ttest_ind(timings[:, 0], timings[:, 1]).pvalue
     median = np.median(timings, axis=0)
     speedup = median[0] / median[1]
+    if args.dump_raw_metrics:
```

Comment:
why are we saving to numpy file?

Reply:
we can dump to json files if you prefer that way?

Reply:
having raw metric numbers can help us estimate the uncertainties in timing measurements

Reply:
Are you planning to write scripts to analyze this?  What analysis are you planning to do?

We could also just add more metrics to the `.csv` already generated.

Reply:
We do have scripts to analyze this. We currently use standard deviation plus vertical error bars to estimate the uncertainty.

We can add those metrics to the .csv file, while I also think to get the raw metrics is also important. Besides, this flag is off by default and doesn't have noticeable effect to the script?

Reply:
How would someone else use this if they don't have those scripts?  Could we add the error bars to the output so in the common case we don't need an extra step?